core/src/main/resources/lib/layout/breadcrumbBar.jelly

```diff
@@ -44,8 +44,16 @@ THE SOFTWARE.
               <j:forEach var="anc" items="${request.ancestors}">
                 <j:if test="${h.isModel(anc.object) and anc.prev.url!=anc.url}">
                   <li class="item">
+                    <!-- Workaround to set desired name to first breadcrumb (instead of Jenkins) -->
                     <a href="${anc.url}/" class="${h.isModelWithContextMenu(anc.object)?'model-link':null} inside">
-                      ${anc.object.displayName}
+                      <j:choose>
+                        <j:when test="${anc.object == app}">
+                          ${%dashboard}
```

Comment:
Hint: If the English localization is simple enough and rather unlikely to change, it can be put here directly, saves having to create a "default" properties file.

Stylistic preference only.

Reply:
If I understand correctly, your suggestion would be to use `${%Dashboard}` (title case) and remove the default properties, since it will use the property name as the value, right?

Can do that. Although when the i18n suggestion came along, I liked the idea of having a properties for the breadcrumbs, since I would expect it to eventually have more properties in it. I'll change it to title case and squash commits, but if it is decided to remove it, I'm OK with it.